lib/controllers/tweetController.js

```diff
@@ -0,0 +1,27 @@
+const { Router } = require('express');
+const authenticate = require('../middleware/authenticate');
+const Tweet = require('../models/Tweet');
+
+module.exports = Router()
+  .get('/', authenticate, async (req, res, next) => {
+    try {
+      const tweets = {
```

Comment:
Rather than hardcoding a tweet here, this route should have relied on a getAll method from your Tweet model.